src/main/java/com/faforever/client/remote/domain/inbound/FafInboundMessage.java

```diff
@@ -6,12 +6,11 @@
  * Superclass for all server objects. Server objects are deserialized from a JSON-like string, therefore all field names
  * and types must exactly match to what the server sends.. A server object's concrete type is derived by its command
  * field.
- *
  */
 
-public abstract class FafServerMessage extends ServerMessage {
+public abstract class FafInboundMessage extends InboundMessage {
 
-  protected FafServerMessage() {
+  protected FafInboundMessage() {
     super(MessageTarget.CLIENT);
```

Comment:
Not sure why we still need this.

Reply:
I use this class to set the target for all the faf subclass messages